requirements.txt

```diff
@@ -1 +1 @@
-yt-dlp==2021.12.1
\ No newline at end of file
+setuptools~=60.1.1
```

Comment:
Setuptools shouldn't be here. Yt-dlp still should I believe.

Reply:
The library will work without `Yt-dlp`, better will do it optional:
pip install songbird.py[yt-dlp]
pip install songbird.py[youtube-dl]